src/order-history/__snapshots__/OrderHistoryPage.test.jsx.snap

```diff
@@ -48,20 +61,32 @@ exports[`<OrderHistoryPage /> Renders correctly in various states renders orders
       >
         <button
           aria-label="Next, Page 3"
-          className="btn next page-link"
+          className="next page-link btn btn-primary"
```

Comment:
why are we adding `btn-primary` class?

Reply:
Changes to the snapshot were made through `npm test -- -u` which is the command to update snapshot. These changes are happening due to updated versions of Pragon and other deps.